--- old
+++ new
@@ -11,1 +11,1 @@
-## 2  Easy to use without training and supports quick creating of case 
+## 1 Easy to use without training and supports quick creating of case 
@@ -18,1 +18,8 @@
-   4. 
+   
+> For now we're building the cli version so this ideas will be implemented in the CLI 
+
+Things to start working on to solve this problem 
+
+- create the cli interface
+- create the cli version of the case creation
+- 